source/dub/dub.d

```diff
@@ -1106,8 +1217,9 @@ class Dub {
 		m_defaultCompiler = res.empty ? compilers[0] : res.front;
 	}
 
-	private Path makeAbsolute(Path p) const { return p.absolute ? p : m_rootPath ~ p; }
-	private Path makeAbsolute(string p) const { return makeAbsolute(Path(p)); }
+	private Path makeAbsoluteRelativeToRoot(Path p) const { return p.absolute ? p : m_rootPath ~ p; }
+	private Path makeAbsoluteRelativeToRoot(string p) const { return makeAbsoluteRelativeToRoot(Path(p)); }
+	private static Path makeAbsoluteRelativeToCwd(Path p) { return p.absolute ? p : getWorkingDirectory() ~ p; }
```

Comment:
seemed good to be really explicit what the reference point is for making absolute paths.